src/openforms/authentication/contrib/org_oidc/plugin.py

```diff
@@ -13,6 +13,24 @@
 from ...registry import register
 
 
+def _reverse_plus(name, *, args=None, kwargs=None, request=None, query=None):
```

Comment:
Proposal to make this `_reverse_plus()` a shared util so we have less of this noisy `reverse`/`furl` everywhere (the blacked versions of those lines are terrible)

Reply:
let's do it - you can put it in `openforms.utils.urls` and ensure it's picked up by the developer documentation.

I'd also consider adding a kwarg `make_absolute: bool = False` and if true, pass the result through `build_absolute_uri`